fractal_server/app/models/job.py

```diff
@@ -103,6 +108,8 @@ class ApplyWorkflow(_ApplyWorkflowBase, SQLModel, table=True):
     )
     workflow: Workflow = Relationship()
 
+    workflow_dump: dict[str, Any] = Field(sa_column=Column(JSON))
```

Comment:
Reminder: can you add the description to the docstring?

Reply:
sure